lib/components/clipboard-button/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactDom from 'react-dom';
+import Clipboard from 'clipboard';
+
+function ClipboardButton({ text, disabled }) {
```

Comment:
I noticed that we have this prop but we never set it. Should we just leave it out for now?